tests/spec/fs.write.spec.js

```diff
@@ -5,6 +5,18 @@ var expect = require('chai').expect;
 describe('fs.write', function() {
   beforeEach(util.setup);
   afterEach(util.cleanup);
+  
+  // the file is underfined, will create problem
```

Comment:
Typo nit: 'unde**r**fined' might be intended to spell 'undefined'.

Reply:
Thank you for telling me, I will fix the problem late.